packages/fx-core/src/plugins/resource/apiconnector/utils.ts

```diff
@@ -39,3 +41,69 @@ export function checkInputEmpty(inputs: Inputs, ...keys: string[]) {
     }
   }
 }
+
+export class Notification {
+  public static GetBasiString(apiName: string, components: string[], languageType: string): string {
```

Comment:
We can keep current design. In the future, we can see how to let each auth type's (and programming language's) template provider to provide the message. Since the content highly depends on the template.

Reply:
sure